caravel/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -39,16 +39,6 @@ class TabbedSqlEditors extends React.Component {
       window.history.replaceState({}, document.title, this.state.cleanUri);
     }
   }
-  getQueryLink(qe) {
-    const params = [];
-    if (qe.dbId) params.push('dbid=' + qe.dbId);
-    if (qe.title) params.push('title=' + qe.title);
-    if (qe.schema) params.push('schema=' + qe.schema);
-    if (qe.autorun) params.push('autorun=' + qe.autorun);
-    if (qe.sql) params.push('sql=' + qe.sql);
-
-    return getLink(this.state.cleanUri, params);
```

Comment:
I'm trying to understand the new logic and I don't see where the url gets crafted before sending to the backend for shortening.